frontend/src/employee-mobile-frontend/components/messages/ThreadView.tsx

```diff
@@ -150,6 +150,7 @@ const RoundIconButton = styled.button`
 
 const MessageContainer = styled.div`
   border-radius: ${defaultMargins.s};
+  max-width: 90%;
```

Comment:
Is this to prevent the `kasdjfoashkdasdhasdhkaskdjaskhjdaskjfsjofashjodasjkasjkhdasfhdj` overflow? In that case, `overflow-wrap: anywhere;` could be considered too.